ACCESSIBILITY.md

```diff
@@ -0,0 +1,90 @@
+2. **Sharing openly**: We aim to draft, write, review, and edit our open access guides using open source practices. We do this in order to enable people from a broad range of backgrounds to get involved in the project.
+3. **Supporting inclusive participation**: We acknowledge that openness alone does not make a project accessible in a broad sense, and we are committed to maintaining a socio-technical infrastructure that enables inclusive participation more broadly. This includes hosting spaces for contribution, documenting our practices, and actively gathering feedback.
+
+The practices and policies listed here relate to these three operating principles, which shape how we work within the community to enable access on the web, on our community platforms, and while planning events.
+
+### 2.0 Community Accessibility Practices
+
+As an open source community that relies on projects and platforms for our underlying infrastructure, we acknowledge that _The Turing Way_ is limited in what standalone access-centered support we can offer.
+
+With this being said, we are committed to ensuring greater access to our guides, events, and community platforms through a variety of ways. 
+
+1. **Ensuring web accessibility for our open access resources**:  Our project is built with [JupyterBook](https://jupyter-accessibility.readthedocs.io/en/latest/index.html) on the [Github](https://accessibility.github.com/) platform, which have initiatives that address their compliance with [W3C WAI's Web Content Accessibility Guidelines 2.0](https://www.w3.org/TR/WCAG/). As a community documentation project, we aim to use and employ appropriate and relevant [alt text (also known as 'alternative text'](https://book.the-turing-way.org/community-handbook/accessibility/alt-text) for all of our images, charts, and data visualisations.
+
+2. **Enabling accessibility of our shared community platforms**: Our community is dedicated to enabling access across the various platforms we communicate on, which extends not only to the accessibility practices mentioned above, but also to the language we employ to ensure a broad audience. Our [slack welcome guide](https://the-turing-way.netlify.app/community-handbook/communication-channels/slack-welcome-guide.html) delineates best practices we use on the platform.
+
+3. **Implementing accessible practices for community events**: As a community, we frequently organise events for a variety of purposes: for coworking, trainings, workshops, panels, conference talks, and more. We are committed to supplying and recording audio transcripts and closed captioning, uploading slides and/or presentation materials, and enabling broad access to our program content [as described in our community handbook](https://book.the-turing-way.org/community-handbook/accessibility/event-organising).
```

Comment:
```suggestion
1. **Ensuring web accessibility for our open access resources**:  Our project is built with [JupyterBook](https://jupyter-accessibility.readthedocs.io/en/latest/index.html) on the [Github](https://accessibility.github.com/) platform, which have initiatives that address their compliance with [W3C WAI's Web Content Accessibility Guidelines 2.0](https://www.w3.org/TR/WCAG/). As a community documentation project, we aim to use and employ appropriate and relevant [alt text (also known as 'alternative text'](https://book.the-turing-way.org/community-handbook/accessibility/alt-text) for all of our images, charts, and data visualisations.
2. **Enabling accessibility of our shared community platforms**: Our community is dedicated to enabling access across the various platforms we communicate on, which extends not only to the accessibility practices mentioned above, but also to the language we employ to ensure a broad audience. Our [slack welcome guide](https://the-turing-way.netlify.app/community-handbook/communication-channels/slack-welcome-guide.html) delineates best practices we use on the platform.
3. **Implementing accessible practices for community events**: As a community, we frequently organise events for a variety of purposes: for coworking, trainings, workshops, panels, conference talks, and more. We are committed to supplying and recording audio transcripts and closed captioning, uploading slides and/or presentation materials, and enabling broad access to our program content [as described in our community handbook](https://book.the-turing-way.org/community-handbook/accessibility/event-organising).
```

With the linebreaks between each item this would render as three lists rather than a single list.